lib/vendor.rb

```diff
@@ -1,3 +1,30 @@
 class Vendor
+    attr_reader :name
+
+    attr_accessor :inventory
```

Comment:
This could probably exist as an attr_reader. You're not going to update this value directly outside of this class.